third_party/move/move-core/types/src/value.rs

```diff
@@ -185,6 +186,8 @@ impl MoveValue {
     }
 }
 
+pub type BytesWithLayout = (Bytes, Option<MoveTypeLayout>);
```

Comment:
If we are really setting Layout only for aggregators, and not always, maybe we should name this BytesWithAggregatorLayout or have a  very clear comment here

Reply:
Done.